docs/02_getting-started/02_development-environment/06_start-your-node-setup.md

```diff
@@ -54,6 +54,8 @@ These settings accomplish the following:
 
 ### Troubleshooting
 
+After entering `keosd &`, if you encounter a message saying "3120000 wallet_exception: Wallet exception Failed to lock access to wallet directory; is another keosd running?", you may have already had `keosd` processes running in the background. Try killing them by, for example, `pkill keosd`.
```

Comment:
@b1aafulei 

Thanks for contributing to the docs! Yes, it would be nice to have this troubleshooting note in `keosd`. 

I reviewed the PR and see that the best location to insert this note would be in **Step 1.1: Start keosd**. Also, I have minor edit suggestions to the text. Would it be okay if I directly commit to your branch? Or would you prefer a separate PR with my edits?

Reply:
Unable to see b1aafulei's reply here but he gave me a go-ahead to directly commit to his branch.